libr/core/cconfig.c

```diff
@@ -853,6 +853,23 @@ static bool cb_emuskip(void *user, void *data) {
 	return true;
 }
 
+static bool cb_jsonencoding(void *user, void *data) {
+	RConfigNode *node = (RConfigNode*) data;
+	if (*node->value == '?') {
+		if (strlen (node->value) > 1 && node->value[1] == '?') {
+			r_cons_printf ("choose either: \n"\
+			"none (default)\n" \
+			"base64 - encode the json string values as base64\n" \
+			"bytearray - convert the string to hexpairs\n" \
```

Comment:
i would say "hex" instead of bytearray.

also i expect also the type: array of numbers as encoder for that for example:

pj_ke (pj, "word", "foo", 3);

will result in:

{"word": [102, 102, 111]}

Reply:
I added both, personally, I would like to use a hexpair string, because then i dont need to do extra work on the python side (e.g. using r2pipe) to rebuild the string. But I also think it makes sense to encode it as an array like you said, so there is both :)